chainer_chemistry/dataset/parsers/csv_file_parser.py

```diff
@@ -13,6 +13,18 @@
 import traceback
 
 
+def _read_csv(filepath):
+    try:
+        # It is recommended to use `read_csv` method in pandas version
+        # after 0.18.x
+        df = pandas.read_csv(filepath)
+    except AttributeError as e:
+        # It is deprecated in newer versions of pandas, but we use
+        # this method for older version of pandas.
+        df = pandas.DataFrame.from_csv(filepath)
+    return df
```

Comment:
It is not directly related to this PR but should we write a policy somewhere that we do not test older versions of dependent modules (like Pandas) except Python itself, although we sometimes do make code work with them, as we do here?

Reply:
I imagined to make a consensus within a development team and write it somewhere in the [development policy](http://chainer-chemistry.readthedocs.io/en/stable/development.html).